src/test_helpers/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 The Exonum Team
+// Copyright 2019 The Exonum Team
```

Comment:
Can this module be moved to the `tests` folder?

Reply:
This module may be useful if you want to integrate your own service with anchoring and also, additional modules in `tests` subdir are painful.

Reply:
Maybe then wrap the module contents into `#[cfg(test)]`?

Reply:
This one does not work for external tests.